xtras/src/send_async_safe.rs

```diff
@@ -47,11 +47,11 @@ where
             return Err(xtra::Error::Disconnected);
         }
 
-        let send_fut = self.send(msg);
+        let rx = self.send(msg).split_receiver().await;
```

Comment:
IMHO this is a very good change - we will send faster, and then only await the result for log purposes in a spawned task.

Reply:
I think the send won't necessarily be faster, but the reason I made the change in here was so that we can see the rest of the spans that are caused by it, thus bringing it in line with the other send_async_safe impls.

Of course, whether the handlers should be children of the span at all is its own discussion.